dio/lib/src/dio_event_processor.dart

```diff
@@ -164,29 +164,8 @@ class DioEventProcessor implements EventProcessor {
 
     return SentryResponse(
       headers: _options.sendDefaultPii ? headers : null,
-      url: response?.realUri.toString(),
-      redirected: response?.isRedirect,
-      body: _getResponseData(dioError.response?.data),
+      bodySize: dioError.response?.data?.length as int?,
```

Comment:
Missing field `cookies`

Reply:
added in https://github.com/getsentry/sentry-dart/pull/1095/commits/fa59ae2357be26055cf25f8601111c57e8c85c2f